react-usercard/src/components/UserCard.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+class UserCard extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+     createUserCard() {
```

Comment:
This is interesting. I'm kind of curious why you chose to create the card this way.

Reply:
I didn't need to- was using this as a reference from my old project. I hadn't cleaned the code up yet.